src/pages/VerifyConfirmation.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {Container, Heading, VStack} from '@chakra-ui/react'
+import {Icon} from "@chakra-ui/icons";
+import {ReactComponent as CheckmarkIllustration} from "../assets/checkmark.svg";
+
+export default function VerifyConfirmation() {
+    return (
+        <VStack minH='100vh' minW='fit-content' p={4} spacing={12} position='relative' justify='center'>
+            <VStack>
+                <Heading>Thank you for registering on group matcher!</Heading>
```

Comment:
Maybe keep the current format of the Heading+Text in the <Verify> component and just change the text inside the container below?

So keep this structure:
```
<VStack>
    <Heading>Thank you, {adminData.name}!</Heading>
    <Text textAlign='center' w='md' color='gray.600'>Your account has been successfully created.</Text>
  </VStack>
```
(perhaps without the name if you don't want to set up the process of passing a state to `navigate(...)`)